package/tests/test_models/test_utils.py

```diff
@@ -0,0 +1,31 @@
+from kedro_viz.models.utils import get_dataset_type, get_file_size
+
+try:
+    # kedro 0.18.11 onwards
+    from kedro.io import MemoryDataset
+except ImportError:
+    # older versions
+    # older versions
+    from kedro.io import MemoryDataSet as MemoryDataset
+
+
+def test_get_dataset_type(dataset=None):
+    assert get_dataset_type(dataset) == ""
+    assert get_dataset_type(MemoryDataset()) == "io.memory_dataset.MemoryDataset"
```

Comment:
What is this test for?

Reply:
Testing utility function get_dataset_type which returns the dataset type as a string based on the dataset input